src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseTransport.php

```diff
@@ -96,7 +96,10 @@ protected function doSend(MessageInterface $message): SentMessage
         $success = $response->toArray(false);
 
         $sentMessage = new SentMessage($message, (string) $this);
-        $sentMessage->setMessageId($success['results'][0]['message_id'] ?? '');
+
+        if (null !== $success['results'][0]['message_id'] ?? null) {
```

Comment:
```suggestion
        if (null !== $messageId = $success['results'][0]['message_id'] ?? null) {
            $sentMessage->setMessageId($messageId);
```

Reply:
I am not a big fan of assigning inside a comparison, plus this variable is not used anymore, so one time variables can be avoided

Reply:
yeah indeed, matter of taste :)